### PR TITLE
Add a sales pitch

### PR DESCRIPTION
**Pull request by smaslennikov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,27 @@
...
+    1. DMZ (public resources such as load balancers)
+    2. Routing (NGINX/HAProxy/layer 7 routing rules)
+    3. Services (for backend web services, async workers, and ETL jobs)
+    4. Data (for databases: e.g. Postgres, RabbitMQ, Elasticsearch, basically things that are stateful)
+    5. Admin (for administrative services like ZooKeeper and etcd and logstash and nagios and whatever)
+
+
+### Rationale/Justification:
+
+This network framework is known as Titan and has been deployed for large enterprise customers. Rather than building a network from scratch, this network framework was used to provide a scalable and secure network layout for the PoC. At you company’s scale, it is likely that a more fine-grained solution would be advisable; seeing as this SoW was for a proof-of-concept, Titan was chosen to accelerate implementation.
```
Comment (naftulikay) [inline README.md]: things to fix:

 - `you` :point_right: `your`
 - drop `SoW` and `PoC`

Maybe just drop the whole rationale thing.

File: README.md [hunk 2]
```diff
@@ -2,6 +2,27 @@
...
+    1. DMZ (public resources such as load balancers)
+    2. Routing (NGINX/HAProxy/layer 7 routing rules)
+    3. Services (for backend web services, async workers, and ETL jobs)
+    4. Data (for databases: e.g. Postgres, RabbitMQ, Elasticsearch, basically things that are stateful)
+    5. Admin (for administrative services like ZooKeeper and etcd and logstash and nagios and whatever)
+
+
+### Rationale/Justification:
+
+This network framework is known as Titan and has been deployed for large enterprise customers. Rather than building a network from scratch, this network framework can be used to provide a scalable and secure network layout for the PoC.
```
Comment (naftulikay) [inline README.md]: what does the `PoC` mean in this context?

Comment (smaslennikov) [reply README.md]: any poc the reader may have in mind

Comment (smaslennikov) [reply README.md]: or idk

**Comment by naftulikay** [comment]
Travis failed because the AWS Terraform provider apparently deprecated certain things. We should fix in the future.

**Pull request merged**